README.md

```diff
@@ -90,20 +77,25 @@ invoked.
 
 ### HMAC
 
-Create a `Hmac` object with the hash function and secret key being
-used.  The `Hmac` object is then used like the other hash calculating
-objects.
+Create an instance of the [`Hmac`][Hmac] class with the hash function
+and secret key being used.  The object can then be used like the other
+hash calculating objects.
 
 ```dart
-  ...
+import 'dart:convert';
+import 'package:crypto/crypto.dart';
+import 'package:crypto/src/digest_sink.dart';
 
-  var keyBytes = UTF8.encode(keyString);
-  var bytes = UTF8.encode(dataString);
+void main() {
+  var keyBytes = UTF8.encode('p@ssw0rd');
+  var bytes = UTF8.encode("foobar");
 
   var hmac = new Hmac(sha256, keyBytes); // HMAC-SHA256
   var digest = hmac.convert(bytes);
-  print("Digest as bytes: ${digest.bytes}");
-  print("Digest as a hexadecimal string: $digest");
+  
+  print("HMAC digest as bytes: ${digest.bytes}");
+  print("HMAC digest as hex string: $digest");
+}
 ```
 
 ## Test status
```

Comment:
Can you remove this and everything below it? It's inaccurate and redundant at this point.